coverage/htmlfiles/coverage_html.js

```diff
@@ -172,7 +172,10 @@ coverage.index_ready = function ($) {
     // Look for a localStorage item containing previous sort settings:
     var sort_list = [];
     var storage_name = "COVERAGE_INDEX_SORT";
-    var stored_list = localStorage.getItem(storage_name);
+    var stored_list = undefined;
+    try {
```

Comment:
Note - didn't look like there was any better way of testing this as merely accessible the variable caused the error. So, try/catch seemed to be the only option